### PR TITLE
Allow 0.001 float value

### PR DESCRIPTION
**Pull request by yannl35133**
Such float values with leading zeroes were rejected.

### CODE REVIEW

**Comment by yannl35133** [comment]
#652

**Comment by yannl35133** [comment]
Seems like why3 is not properly recognized, I doubt it has anything to do with my modifications and don't know how to fix it

File: src/parsing/lpLexer.ml [hunk 1]
```diff
@@ -108,7 +108,7 @@ exception SyntaxError of strloc
 let digit = [%sedlex.regexp? '0' .. '9']
 let nonzero_nat = [%sedlex.regexp? '1' .. '9', Star digit]
 let nat = [%sedlex.regexp? '0' | nonzero_nat]
-let float = [%sedlex.regexp? nat, '.', Opt (nat)]
+let float = [%sedlex.regexp? Opt nat, '.', Opt (Star digit)]
```
Comment (fblanqui) [inline src/parsing/lpLexer.ml:111]: Instead, I propose to use:
```
let float = [%sedlex.regexp? nat, '.', Plus digit]
```

**Comment by fblanqui** [comment]
I seems that alt-ergo is not installed while it is needed for running tests. @gabrielhdt shouldn't tests/dune include alt-ergo as dependency?

**Pull request merged**